### PR TITLE
I18N: Add script to automatically generate scummvm.appdata.xml from .po files

### PR DESCRIPTION
**Pull request by ankushdutt**
This PR adds a script to support multilanguage in scummvm.appdata.xml. The multilanguage data is extracted from po/*.po files.

Related Discord discussion: https://discord.com/channels/581224060529148060/1086942326489895033/1086976457311531058

### CODE REVIEW

File: dists/scummvm.appdata.xml [hunk 1]
```diff
@@ -6,29 +6,258 @@
   <project_license>GPL-3.0-or-later</project_license>
   <name>ScummVM</name>
   <summary>Interpreter for numerous adventure games and role-playing games</summary>
+  <summary xml:lang="ru">Интерпретатор множества приключенческих и ролевых игр</summary>
```
Comment (sev-) [inline dists/scummvm.appdata.xml:9]: These, and other language tags should be sorted alphabetically, so their sequence stays consistent

Comment (ankushdutt) [reply dists/scummvm.appdata.xml]: Fixed

File: devtools/generate-appdata.py [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+    par2 = par2.replace('"', '')
+    p_elem = doc.createElement("p")
+    p_text = doc.createTextNode(par2)
+    p_elem.appendChild(p_text)
+    lang = file[0] + file[1]
+    p_elem.setAttribute("xml:lang", lang)
+    description_elem.appendChild(p_elem)
+
+# paragraph 3
+p_elem = doc.createElement("p")
```
[the remark this answers is not in the document]
Comment (ankushdutt) [reply devtools/generate-appdata.py]: Fixed and changed the code to have an in-source copy of the template and added code to substitute the specific keywords with relevant translations.

**Comment by sev-** [comment]
Thank you!

**Pull request merged**